src/logger.c

```diff
@@ -321,7 +321,7 @@ logger_init(char *file, char *domains, int severity)
       return -1;
     }
 
-  ret = fchown(fileno(logfile), runas_uid, 0);
+  ret = fchown(fileno(logfile), runas_uid, -1);
```

Comment:
What is the effect of this change? This should at least be a separate commit and better a separate pull request.